js/src/Image.js

```diff
@@ -23,17 +23,19 @@ var ImageModel = widgets.DOMWidgetModel.extend({
 //var action = firefly.action;
 
 
-var seq = 1;
 // Custom View. Renders the widget model.
 var ImageView = widgets.DOMWidgetView.extend({
     render: function() {
         this.url = this.model.get('url');
-        this.el.id = `imageViewer-${seq++}`;
+        this.el.id = this.model.get('plot_id');
         // disable Jupyter notebook keyboard manager
         // shortcut handling prevents input into dialog fields
         this.el.onclick = () => {
             Jupyter.keyboard_manager.disable();
         };
+        this.model.set('conn_id', String(firefly.util.getWsConnId()));
+        this.model.set('channel', String(firefly.util.getWsChannel()));
```

Comment:
So, at the moment we always use the default channel id, based on user_key. Do we want to give widget users an ability to pass channel id?

I have noticed that the header for channel is wrong in firefly_client. It should be `FF-channel`. Currently it is `FF_FF-channel`. This might explain channel-related communication issues we've had before.